docs/libraries/azure-core/reference/data-types.md

```diff
@@ -247,6 +279,12 @@ Supported versions of Azure.Core TypeSpec building blocks.
 enum Azure.Core.Versions
 ```
 
+### `PollingOptionKind` {#Azure.Core.PollingOptionKind}
+
+```typespec
+union Azure.Core.PollingOptionKind
```

Comment:
Could be a docs bug, but why doesn't it show the values?

Reply:
It doesn't go intodetail on model, enum, union types.